distribution/docs/src/main/resources/_contents/_endpoints/content-endpoint-contents.adoc

```diff
@@ -0,0 +1,536 @@
+
+==== Content Endpoint
```

Comment:
We no longer have the Content Endpoint or the Content Framework so you should be able to remove this entire page but may want to check with @stustison on any documentation that should be moved over to the Catalog REST endpoint. 

I'm also not seeing the Catalog REST endpoint in this PR, am I missing it?